lipton_custom/fixtures/custom_field.json

```diff
@@ -78,19 +78,19 @@
   "in_global_search": 0,
   "in_list_view": 0,
   "in_preview": 0,
-  "in_standard_filter": 1,
-  "insert_after": "garden_mark",
+  "in_standard_filter": 0,
+  "insert_after": "grade",
   "is_system_generated": 0,
   "is_virtual": 0,
-  "label": "Grade",
+  "label": "Column Break",
```

Comment:
Gotta keep the labels empty for Column Break fieldtypes

Reply:
Updated in new PR